libraries/AP_HAL_SITL/Storage.h

```diff
@@ -5,19 +5,19 @@
 #include "AP_HAL_SITL_Namespace.h"
 #include <AP_FlashStorage/AP_FlashStorage.h>
 
-// define which storage system to use. This allows us to test flash storage with --sitl-flash-storage
-// configure option
-#ifndef STORAGE_USE_FLASH
-#define STORAGE_USE_FLASH 0
-#endif
-
-#define STORAGE_USE_POSIX 1
-
 #define STORAGE_LINE_SHIFT 3
 
 #define STORAGE_LINE_SIZE (1<<STORAGE_LINE_SHIFT)
 #define STORAGE_NUM_LINES (HAL_STORAGE_SIZE/STORAGE_LINE_SIZE)
 
+#ifndef STORAGE_USE_FLASH
+#define STORAGE_USE_FLASH 1
```

Comment:
shouldn't this remain 0 by default

Reply:
I reckon it is reasonable to build both in by default - this is SITL-only after all.  It means I can test both mechanisms without rebuilding the firmware.